src/api/app/services/scm_exception_message.rb

```diff
@@ -49,6 +49,8 @@ class ScmExceptionMessage
   }.freeze
 
   GITLAB_EXCEPTIONS = {
+    Gitlab::Error::BadGateway =>
+      'GitLab is not responding.',
```

Comment:
At some point, we could unify the messages. For GitHub we have

```
    Octokit::BadGateway =>
      'Bad gateway. Please try again later.'
```
But I understand this is not the scope of this PR.

Reply:
Is it not?

Reply:
I changed the text, to use the same as we had for GitHub. Please have a look again.